blocks/library/text/index.js

```diff
@@ -62,6 +77,7 @@ registerBlockType( 'core/text', {
 				} }
 				onMerge={ mergeBlocks }
 				style={ { textAlign: align } }
+				className={ `drop-cap-${ dropCap }` }
```

Comment:
Indeed, plan is to turn it into `has-drop-cap`. I am not sure if it belong in editable, it feels like purely a text/paragraph addition. (Makes no sense in quotes, captions, etc.)